pageStyles/login.module.scss

```diff
@@ -0,0 +1,21 @@
+﻿$font-size: 60px;
+$font-family: sans-serif;
+$font-color: orange;
+
+.h1 {
+  font-size: $font-size;
+  font-family: $font-family;
+  color: $font-color;
```

Comment:
It would be good to give the font size and colour more meaningful names once they're pulled out into a shared stylesheet.
Something like `$header-font-size`, `$testswitch-yellow` - whatever you think makes the most sense.